neo/Consensus/ConsensusContext.cs

```diff
@@ -210,14 +210,44 @@ public ConsensusPayload MakePrepareRequest()
         {
             byte[] buffer = new byte[sizeof(ulong)];
             random.NextBytes(buffer);
+            Block.ConsensusData.Nonce = BitConverter.ToUInt64(buffer, 0);
```

Comment:
Why did you move this up? For organization?

Reply:
Yes, when is computed